students/pkoleyni/Lesson03/Slicing_Lab.py

```diff
@@ -0,0 +1,49 @@
+def exchange_first_last(seq):
+    """
+    :param seq: any python seq, like String,List,Tuple
+    :return:
+    """
+    return (seq[-1:]+seq[1:-1]+seq[0])
+
+def every_other_item(seq):
+    return (seq[0::2])
```

Comment:
you don't need parens with return statement, it's not a function call -> `return seq[0::2]`

also, it can just be `return seq[::2]` as 0 is a default value